# Tamaño de muestra


Determinamos el tamaño de muestra necesario para estimar una proporción con un margen de error dado.


Usamos la fórmula del tamaño de muestra para proporciones con un nivel de confianza del 95%.


In [1]:

import math
Z = 1.96  # 95% de confianza
p = 0.5   # proporción esperada
e = 0.05  # margen de error permitido
n = (Z**2 * p * (1-p)) / (e**2)
print('Tamaño de muestra aproximado:', math.ceil(n))


Tamaño de muestra aproximado: 385


Para una proporción esperada del 50% y un margen de error del 5%, se requiere una muestra de aproximadamente 385 observaciones.


**Ejercicio:** Cambia el margen de error a 3% y calcula el nuevo tamaño de muestra. *Hint:* ajusta `e = 0.03`.
